app/controllers/services/service_instances_controller.rb

```diff
@@ -175,6 +175,33 @@ def permissions(guid)
       end
     end
 
+    get '/v2/service_instances/:guid/service_keys', :service_keys
```

Comment:
Hi @jeffh, this is Tom from IBM Beijing lab, thanks for your comments, one question is do we have tests that verify the `define_attributes`? if no, can we just remove the whole method and add the `to_many` relationship and resubmit this PR?